ReplaceServiceBusConnectionString.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding()]
+Param(
+   [Parameter(Mandatory=$True,Position=1)]
+   [string]$directoryPath,
+	
+   [Parameter(Mandatory=$True)]
+   [string]$connectionString
+)
+
+$configFiles = Get-ChildItem -Path $directoryPath -Recurse  -Include *.config
+$configFiles | ForEach-Object {
+        $configFile = $_.FullName
+
+        $xmlFile = (Get-Content $configFile) -as [Xml]
+
+        foreach($add in $xmlFile.configuration.appSettings.add) {
+            if ($add.key -eq 'ServiceBus') 
+            {                                
+                write-host "File Changed : " $configFile
```

Comment:
it's good to print "File Changed" after actually changing it.

Reply:
done